packages/exporter-trace-otlp-http/src/OTLPExporterBase.ts

```diff
@@ -102,6 +112,32 @@ export abstract class OTLPExporterBase<
     });
   }
 
+  private _configureTimeout(timeoutMillis: number | undefined): number {
```

Comment:
is it feasible to write any unit test for this logic?

Reply:
My thought was that since the method is private, we couldn't test it. I did test the logic locally in one of my sample apps.

Reply:
Yeah, it's not very easy.

Since these functions are private methods in the class, it's a bit complicated. They can actually be static functions (they don't access the `this` parameter) and maybe be moved to utils. This way it's trivial to test them, and I think it's common in other parts of the project to have these kinds of functions as `utils`.

Another alternative, which will also validate the timeout logic itself, would be to write tests similar to the existing ones, with `fakeRequest` that takes long time to reply and let a short (1 ms) timeout hit. Then we can verify the promise is rejected with the proper message.

Reply:
Ok. I see what you mean by moving those methods to `utils` and testing it. I will do that. 

And I will test the timeout logic with the `fakeRequest` test.

Reply:
I moved the functions around configuring timeout to the `utils` and added unit testing for that logic [here](https://github.com/open-telemetry/opentelemetry-js/blob/88660355303012cda492b9842492516d5369b33a/packages/exporter-trace-otlp-http/test/common/utils.test.ts#L55).